api/HTMLHyperlinkElementUtils.json

```diff
@@ -0,0 +1,750 @@
+              "notes": "From Gecko 22 to Gecko 44, this property was on the <code>URLUtils</code> mixin. It has been moved either on the <code>HTMLHyperlinkElementUtils</code> mixin, or directly on the interface."
+            },
+            "firefox_android": {
+              "version_added": "22",
+              "notes": "From Gecko 22 to Gecko 44, this property was on the <code>URLUtils</code> mixin. It has been moved either on the <code>HTMLHyperlinkElementUtils</code> mixin, or directly on the interface."
+            },
+            "ie": {
+              "version_added": false,
+              "notes": "In Internet Explorer 9, the host of an <a href='https://developer.mozilla.org/docs/Web/HTML/Element/a'><code><a></code></a> always include the port (e.g. <code>developer.mozilla.org:443</code>), even if there is no explicit port in the <code>href</code> attribute value."
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "hostname": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLHyperlinkElementUtils/hostname",
+          "support": {
+            "webview_android": {
+              "version_added": true,
+              "notes": "Starting in Chrome 52, this property was moved to <a href='https://developer.mozilla.org/docs/Web/API/URL'>URL</a>"
+            },
+            "chrome": {
+              "version_added": true,
+              "notes": "Starting in Chrome 52, this property was moved to <a href='https://developer.mozilla.org/docs/Web/API/URL'>URL</a>"
+            },
+            "chrome_android": {
+              "version_added": true,
+              "notes": "Starting in Chrome 52, this property was moved to <a href='https://developer.mozilla.org/docs/Web/API/URL'>URL</a>"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "22",
+              "notes": "From Gecko 22 to Gecko 44, this property was on the <code>URLUtils</code> mixin. It has been moved either on the <code>HTMLHyperlinkElementUtils</code> mixin, or directly on the interface."
+            },
+            "firefox_android": {
+              "version_added": "22",
+              "notes": "From Gecko 22 to Gecko 44, this property was on the <code>URLUtils</code> mixin. It has been moved either on the <code>HTMLHyperlinkElementUtils</code> mixin, or directly on the interface."
```

Comment:
It's been decided not to mention Gecko. Please change "From Gecko 22 to Gecko 44..." to From Firefox 22 to Firefox 44..."